### DISCUSSION TITLE
Will it support Nuxt3?

### DISCUSSION THREAD

**Discussion by evanlong-me** [unovue/reka-ui#332] [Ideas]
I would love to see it natively supported. 🤗

**Comment by zernonia** [comment]
It already is!  😁

https://www.radix-vue.com/overview/installation.html#nuxt-modules (updated)

**Reply by ojvribeiro** [reply]
It seems this section was removed 😟

**Reply by zernonia** [reply]
Oppss.. I've orgrnized the page.
Check this out https://www.radix-vue.com/overview/installation.html#nuxt-modules

**Reply by ojvribeiro** [reply]
Nice!

So, if all components are ready, does it mean this lib is production ready already? Or should we wait a little bit more?

**Reply by zernonia** [reply]
The components are quite stabe now. We have testing writen too. If you are on a side-project, definitely try it out! 😁

**Reply by ojvribeiro** [reply]
Oh, I'll definitely use it in my side project haha. But it is for a big platform of the company I work. We have a lot of custom components and it would be nice to have something like Radix Vue.
